### DISCUSSION TITLE
请问文件命名格式可用参数有哪些

### DISCUSSION THREAD

**Discussion by cicur** [the1812/Bilibili-Evolved#3140] [提问 / Q&A] [answered]
如题，请问文件命名格式可用参数有哪些
我记得在v1的时候是有提示的，但v2中并没有提示，文档中也没有找到相关内容
另外，使用`[title]`参数时文件名内有`电影-高清独家在线观看-bilibili-哔哩哔哩`是正常现象吗，能否把这些去掉，谢谢

**Comment by the1812** [comment, marked as answer]
确实忘加提示了, 而且电影的标题也不对
